TOPICS.txt

```diff
@@ -63,11 +63,18 @@ Parsing
 Pattern recognition
 Performance
 Randomness
-Reactive programming
 Refactoring
 Regular Expressions
 Searching
 Security
 Sorting
 Text formatting
 Transforming
+
+Programming Style
+---
+Behavior driven development
+Functional programming
+Object oriented programming
```

Comment:
According to [Wikipedia](https://en.wikipedia.org/wiki/Object-oriented_programming), it is written with a dash: Object-oriented programming